pkg/windows/windows.go

```diff
@@ -196,6 +198,7 @@ func New(workerIgnitionEndpoint, vxlanPort string, instanceInfo *instance.Info,
 	return &windows{
 			interact:               conn,
 			workerIgnitionEndpoint: workerIgnitionEndpoint,
+			clusterDNSIP:           clusterDNSIP,
 			vxlanPort:              vxlanPort,
```

Comment:
We are slowing expanding on cluster information. This calls for it to be encapsulated into a struct otherwise there is potential for this method signature to keep growing. 
PS: not in this PR but a follow up PR.

Reply:
Agree, will take it as a follow-up action item.

Reply:
@jrvaldes just for tracking purpose , did we have ticket in backlog for follow up ?

Reply:
I have the follow-up PR half-baked, don't feel there is a need to track it with a backlog ticket. Still, I'm fine creating a tech-debt one if that is the general agreement.

Reply:
I doubt the follow up PR will be an easy merge 😉 This needs some thought and is not just about collecting everything into a struct. For instance do we need to overload the struct with all this information just so that we can use it in `runBootstrapper()`? So I suggest a follow up story.

Reply:
Created [WINC-704](https://issues.redhat.com/browse/WINC-704)